src/dbus/server/dbus_thread_object.cpp

```diff
@@ -88,6 +91,16 @@ static uint64_t ConvertOpenThreadUint64(const uint8_t *aValue)
     return val;
 }
 
+static otExtendedPanId Uint64ToExtendedPanId(uint64_t aExtPanId)
+{
+    otExtendedPanId extPanId;
+    uint64_t *      pExtPanId = reinterpret_cast<uint64_t *>(&extPanId.m8[0]);
```

Comment:
It's possible that `extPanId` is not aligned properly.

Use `alignas` when defining `extPanId`, or use `memcpy` instead.